docs/instruments/bmi.json

```diff
@@ -0,0 +1,532 @@
+          "readonly": true,
+          "hideInSurvey": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      }
+    },
+    "helpers": {
+      "noteUnits": {
+        "type": "statictext",
+        "description": {
+          "en-CA": "Note: Please enter measurements in Standard OR Metric units"
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "sectionStandard": {
+        "type": "section",
+        "description": {
+          "en-CA": "Standard units"
+        },
+        "options": {
+          "order": [
+            "groupHeightFeet",
+            "groupHeightInches",
+            "groupWeightLbs"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "sectionMetric": {
+        "type": "section",
+        "description": {
+          "en-CA": "Metric units"
+        },
+        "options": {
+          "order": [
+            "groupHeighCms",
+            "groupWeightKgs"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupHeightFeet": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your height (feet) :"
+        },
+        "options": {
+          "order": [
+            "height_feet",
+            "height_feet_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupHeightInches": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your height (inches) :"
+        },
+        "options": {
+          "order": [
+            "height_inches",
+            "height_inches_status"
+
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupWeightLbs": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your weight (lbs) :"
+        },
+        "options": {
+          "order": [
+            "weight_lbs",
+            "weight_lbs_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "groupHeightCms": {
+        "type": "group",
+        "description": {
+          "en-CA": "Your height (cms) :"
+        },
+        "options": {
+          "order": [
+            "height_cms",
+            "height_cms_status"
+          ],
+          "showDesc": true
+        },
+        "rules": {
+          "requireIf": "",
```

Comment:
How do "requireIf" rules on Groups interact with "requireIf" rules on elements?   Does one override the other?

Reply:
a field's rule would be checked first. if the rule is violated, the group's rule would be checked if it exists